SPECS/intel-desktop-virtualization-k3s/intel-desktop-virtualization-k3s.spec

```diff
@@ -0,0 +1,36 @@
+Name:           intel-desktop-virtualization-k3s
+Version:        0.1
+Release:        1%{?dist}
+Vendor:         Intel Corporation
+Distribution:   Edge Microvisor Toolkit
+URL:            https://github.com/open-edge-platform/edge-desktop-virtualization
+Summary:        Provides Kubevirt (enabled with GTK libarary support and Intel SR-IOV patched QEMU in Virt-Launcher) and IDV Device Plugin for enabling support of local GTK display using pre-built container tar files
+
+License:        Apache-2.0
+Source0:        https://github.com/open-edge-platform/edge-desktop-virtualization/releases/download/pre-release-v0.1/intel-idv-kubevirt-v0.1.tar.gz
+Source1:        https://github.com/open-edge-platform/edge-desktop-virtualization/releases/download/pre-release-v0.1/intel-idv-device-plugin-v0.1.tar.gz
+BuildArch:      x86_64
+Requires:       k3s
+
+%description
+Provides Kubevirt (enabled with GTK libarary support and Intel SR-IOV patched QEMU in Virt-Launcher) and IDV Device Plugin for enabling support of local GTK display using pre-built container tar files
+
+%prep
+
+%build
+
+%install
+mkdir -p %{buildroot}%{_datadir}/%{name}
```

Comment:
Save all required container images with "docker save <image>" and the tar file should be stored under %{buildroot}%{_sharedstatedir}/rancher/k3s/agent/images.

Also, k8s manifest to deploy the plugin should be placed under %{buildroot}%{_sharedstatedir}/rancher/k3s/server/manifests/.

If it's Helm chart, place the chart tgz file under %{buildroot}%{_sharedstatedir}/rancher/k3s/server/static/